lib/SILGen/SILGen.cpp

```diff
@@ -1016,6 +1016,12 @@ void SILGenModule::
 emitStoredPropertyInitialization(PatternBindingDecl *pbd, unsigned i) {
   const PatternBindingEntry &pbdEntry = pbd->getPatternList()[i];
   auto *var = pbdEntry.getAnchoringVarDecl();
+
+  // Nameless vars in interface files should not emit any stored property
+  // initializers.
+  if (var->getBaseName().empty())
```

Comment:
Is it ever correct to emit an initializer for a var parsed from an interface file? You don’t know what the initial value is anyway do you?

Reply:
I think I'm with Slava. If we can avoid doing any of this, we don't have to worry about nameless properties specifically.